packages/core/src/Actor.ts

```diff
@@ -33,15 +36,22 @@ export function createNullActor(id: string): Actor {
 }
 
 /**
- * Creates a null actor that is able to be invoked given the provided
+ * Creates a deferred actor that is able to be invoked given the provided
  * invocation information in its `.meta` value.
  *
  * @param invokeDefinition The meta information needed to invoke the actor.
  */
 export function createInvocableActor<TC, TE extends EventObject>(
-  invokeDefinition: InvokeDefinition<TC, TE>
+  invokeDefinition: InvokeDefinition<TC, TE>,
+  machine: StateMachine<TC, any, TE>
 ): Actor {
   const tempActor = createNullActor(invokeDefinition.id);
+  const serviceCreator = machine.options.services?.[invokeDefinition.src];
+  tempActor.deferred = true;
```

Comment:
what does this property mean for `xstate` itself? right now it is just used by `@xstate/react`

Reply:
Doesn't mean anything for XState - it's just used in v4 as a stopgap.